src/GotDownloader.ts

```diff
@@ -4,12 +4,12 @@ import * as path from 'path';
 
 import { Downloader } from './Downloader';
 
-export class GotDownloader implements Downloader<null> {
-  async download(url: string, targetFilePath: string) {
+export class GotDownloader implements Downloader<any> {
+  async download(url: string, targetFilePath: string, options?: any) {
```

Comment:
You're probably asking, "Why is this `any`?" I'm using the same signature as `got` itself:

https://github.com/sindresorhus/got/blob/f77db9929096d0b56cde7981e3f8b1ca635eb628/source/create.ts#L60